### DISCUSSION TITLE
Is it possible to match Midjourney V6's prompt understanding capabilities?

### DISCUSSION THREAD

**Discussion by IamTirion** [lllyasviel/Fooocus#1547] [Q&A] [answered]
I have seen people use Midjourney V6, and it's very impressive in understanding prompts. The output images pretty much delivered exactly what the prompts described. Meanwhile, my experience with Stable Diffusion is not so good, even in Fooocus. For example, if I write "a man with his hands tied up", it never gives me a man with his hands tied up. Since Midjourney is like the target of Fooocus, I wonder if it's possible to match it in prompt understanding, including with techniques from published researches?

**Comment by ikarsokolov** [comment]
A couple of results for "a man with his hands tied up" prompt with different styles. All Fooocus settings are default, "speed" preset is selected.

Here are some of my thoughts:
- I think **juggernautXL** is default model now. It is not very flexible. Good at straight to the camera realistic photo portraits but lacking in style support and prompt flexibility. I am using [**SDXL-DPO**](https://huggingface.co/mhdang/dpo-sdxl-text2image-v1) model that positions itself as improved base SDXL model. I am not sure about "improved" part but this model is very flexible and has proper support of Fooocus styles.
- **Fooocus V2 (GPT one)** is very strong style. It often drastically changes image composition and style. I suggest to start working with all styles unchecked, find interesting image, fix the seed than start applying styles one by one to check how they affect the picture.

@lllyasviel What do you think about changing default preset model from juggernautXL back to base SDXL, SDXL-DPO or other general purpose model? I think the user who do not seek photorealism will have better initial experience with Fooocus.

![image](https://github.com/lllyasviel/Fooocus/assets/11806/026f866d-8744-4713-87fd-4177de3773ff)

![image](https://github.com/lllyasviel/Fooocus/assets/11806/86da6a79-91db-4387-8367-e398cddc3b78)

![image](https://github.com/lllyasviel/Fooocus/assets/11806/24894f37-802d-4c69-bc0a-26c4981ece8f)

**Comment by lllyasviel** [comment, marked as answer]
We have several ongoing projects related to LLM but not inside project Fooocus.

Unless absolutely necessary, Fooocus itself will not have major revisions, but some other independent projects will come out from us using the same deployment codebase as Fooocus. 

If you like Fooocus, you will also like those incoming projects with very similar Fooocus UI.

**Reply by jepjoo** [reply]
Do you mean another SDXL based project or something different?

**Reply by IamTirion** [reply]
Is it right to say that prompt understanding has more to do with the base model, i.e. SDXL, than the UI, and so there isn't much you can do about it? I think Midjourney V6 has an upgraded model.
